rust/xaynet-mobile/src/ffi/participant.rs

```diff
@@ -345,3 +362,143 @@ pub unsafe extern "C" fn xaynet_ffi_participant_set_model(
         ERR_SETMODEL_MODEL
     }
 }
+
+/// Return the latest global model from the coordinator.
+///
+/// - `buffer` is the array in which the global model should be copied.
+/// - `data_type` specifies the type of the model weights (see [`DataType`]). The C header
+///   file generated by this crate provides an enum corresponding to the parameters: `DataType`.
+/// - `len` is the number of weights the model has
+///
+/// # Return Value
+///
+/// - [`OK`] if the model is set successfully
+/// - [`ERR_NULLPTR`] if `participant` or the `buffer` is NULL
+/// - [`GLOBAL_MODEL_NONE`] if no model exists
+/// - [`ERR_GLOBALMODEL_IO`] if the communication with the coordinator
+/// - [`ERR_GLOBALMODEL_DATATYPE`] if the datatype is invalid
+/// - [`ERR_GLOBALMODEL_LEN`] if the length of the buffer does not match the length of the model
+/// - [`ERR_GLOBALMODEL_CONVERT`] if the conversion of the model failed
+///
+/// # Note
+///
+///    It is **not** guaranteed, that the model configuration returned by
```

Comment:
I think there's an extra indentation level on this paragraph. Not sure if that affects how it is rendered though.